model.py

```diff
@@ -174,23 +174,23 @@ def wall_collision(self, bordersize: float = None):
                 or not bordersize < self.p1.x < self.p2.x < cwidth)
 
     def _move(self, event: tk.Event) -> None:
-        #TODO: This doc is irrelevant to the actual effect of the method
+        #  TODO: This doc is irrelevant to the actual effect of the method
         """Arrète le joueur si il touche aux murs."""
         if not self.wall_collision():
             self.canvas.moveto(
                 self.sprite,
-                event.x - self.width/2,
-                event.y - self.height/2
-            )
+                event.x - self.width / 2,
+                event.y - self.height / 2
+                )
```

Comment:
```suggestion
                event.x - self.width/2,
                event.y - self.height/2
            )
```
Les opérateurs avec une plus grande priorité sont groupés sans espaces